source/Forthon_builder.py

```diff
@@ -106,6 +110,8 @@
 else:              forthonargs.append('--no2underscores')
 if not writemodules: forthonargs.append('--nowritemodules')
 if timeroutines: forthonargs.append('--timeroutines')
+forthonargs.append('--omppkg {}'.format(omppkg))
```

Comment:
Can you change this so that the omp arguments are only added to forthonargs when they are specified (i.e. not None)?